doc/stdenv/stdenv.chapter.md

```diff
@@ -706,6 +706,9 @@ to `~/.gdbinit`. GDB will then be able to find debug information installed via `
 
 The installCheck phase checks whether the package was installed correctly by running its test suite against the installed directories. The default `installCheck` calls `make installcheck`.
 
+Because this adds overhead to every build, it is usually recommended to add the test to
+<xref linkend="var-meta-tests"/> instead.
```

Comment:
My 2c is that I think installCheck is primarily useful for applications that have test-suites in tree (that maybe rely on build system configuration artifacts), that aren't then shipped as part of the package build.

If you as a package maintainer are defining a test, then it probably does make sense to add it as a separate passthru.tests test.

Reply:
that's a great distinction, updated to make that more clear